gittyup.nim

```diff
@@ -1013,12 +803,16 @@ proc clone*(uri: Uri; path: string; branch = ""): GitResult[GitRepository] =
     var
       options = cast[ptr git_clone_options](sizeof(git_clone_options).alloc)
     try:
-      withResultOf git_clone_options_init(options, GIT_CLONE_OPTIONS_VERSION):
+      #[
+        withResultOf git_clone_options_init(options, GIT_CLONE_OPTIONS_VERSION):
+          discard
+      ]#
+      withResultOf git_clone_options_init(options, 1):
```

Comment:
Can we put a hack in for now, with a `.warning.` so we remember to remove it when hlibgit2 supports the constant value?